### ISSUE TITLE
Some of my icons are wrong

### ISSUE THREAD

**Issue by WhyNotHugo** [Peltoche/lsd#357]
Some of my icons are wrong, and I'm not sure if lsd is mapping incorrectly, or if my patched font has something off (sorry, I'm not sure how to troubleshoot further).

FWIW, most icons work fine (directories, python files, plain files, markdown, images, tarballs). The ones described below are the only exceptions I've found so far.

Screenshot:
![2020-04-21T10:29:51,727920152+02:00](https://user-images.githubusercontent.com/730811/79843828-0e48da80-83bb-11ea-8d06-36cefaa3e95d.png)

Copy-paste (so you know exactly which unicode points `lsd` is showing):
```
➜ lsd docker-compose.yml poetry.lock uwsgi.ini
  docker-compose.yml    poetry.lock    uwsgi.ini
```

The middle one is obviously broken.
The other two just don't match (OpenSUSE icon for docker, and Windows 10 logo for a config file).

I'm using Operator Mono, patched with nerd fonts.

**Comment by WhyNotHugo** [comment]
Did a bit more digging around this. I'd gotten my patched fonts from [this repository](https://github.com/piq9117/operator-mono-nerdfont)(note: paid license needed for it!), but it seems to have some glyphs wrong.

Checking with [Nerd Font](https://www.nerdfonts.com/cheat-sheet)'s cheat sheet, I've confirmed that the above code-points correspond to:

f308: docker
e21e: ruby
f178: right-arrow

So the first one is fine, but the other two seem wrong:
- `poetry.lock` is a python-specific lockfile
- `uwsgi.ini` is also a python-specific config file

Not sure if icons are based on full filename or just the extension -- if it's just the extension, then maybe something more generic is applicable.

I can try and see if I can work up a PR improving this if you thing it makes sense.

**Comment by lights0123** [comment]
This works fine with the FuraCode patched font, your copy/paste shows up with the Docker logo and a gem for the lockfile.

**Comment by WhyNotHugo** [comment]
Oh, yeah. So it seems that the docker one was my font being badly patched. I addressed the other two in #358.
